utils/upx.xml

```diff
@@ -0,0 +1,108 @@
+        <manifest-digest sha256new="VJ42FPTOEC5C6LX5KNB2YTVS7H33W67NS7Y4DA4W2DZ5LCD233VQ"/>
+        <archive extract="upx-3.95-arm_linux" href="https://github.com/upx/upx/releases/download/v3.95/upx-3.95-arm_linux.tar.xz" size="417600" type="application/x-xz-compressed-tar"/>
+      </implementation>
+      <implementation arch="Linux-x86_64" id="sha1new=c76d7e67841828176bc704a980c045561309e6de">
+        <manifest-digest sha256new="BDCVFUHXSYQOIAD7YZFF4A6VL4X7CVFL655XDLSBR7AMQBEA6XPQ"/>
+        <archive extract="upx-3.95-amd64_linux" href="https://github.com/upx/upx/releases/download/v3.95/upx-3.95-amd64_linux.tar.xz" size="456788" type="application/x-xz-compressed-tar"/>
+      </implementation>
+    </group>
+    <group version="3.96">
+      <implementation arch="Windows-x86_64" id="sha1new=f12c7c1db833048fbac3cd6a10585e2a5f83b2da" released="2020-01-22">
+        <manifest-digest sha256new="U232WDRLJZT4XAEKC6RP6GKKXUGW4Q7X7LIPLR2EEBSKYBQ7XWIQ"/>
+        <archive extract="upx-3.96-win64" href="https://github.com/upx/upx/releases/download/v3.96/upx-3.96-win64.zip" size="468845" type="application/zip"/>
+      </implementation>
+      <implementation arch="Windows-i486" id="sha1new=7594c8908140d5e4f71d66b9a56b5aecb5106d24" released="2020-01-22">
+        <manifest-digest sha256new="VQGI7GW4DUIXHNOPT2RKGAFRXYYUGMB4S2AXLKPTDHCRFIIMPDZQ"/>
+        <archive extract="upx-3.96-win32" href="https://github.com/upx/upx/releases/download/v3.96/upx-3.96-win32.zip" size="468018" type="application/zip"/>
+      </implementation>
+      <implementation arch="Linux-ppc64" id="sha1new=34e06d1ad8174604b89961e9e62ed185dfdaf210" released="2020-01-22">
+        <manifest-digest sha256new="HEESRD4OAJ5ZZIFC4QN3LK7U2IZU4CP73ZNVEY4Y27NXN5KL7KHA"/>
+        <archive extract="upx-3.96-powerpc64le_linux" href="https://github.com/upx/upx/releases/download/v3.96/upx-3.96-powerpc64le_linux.tar.xz" size="505260" type="application/x-xz-compressed-tar"/>
+      </implementation>
+      <implementation arch="Linux-ppc" id="sha1new=20c6543888c5b86aec678cc953decbac0c14a0b7" released="2020-01-22">
+        <manifest-digest sha256new="M2SJQETSV4FNP4TZFCRB64DAKRR6NV32GPQFAPDYN753EI3NQBBA"/>
+        <archive extract="upx-3.96-powerpc_linux" href="https://github.com/upx/upx/releases/download/v3.96/upx-3.96-powerpc_linux.tar.xz" size="479488" type="application/x-xz-compressed-tar"/>
+      </implementation>
+      <implementation arch="Linux-i386" id="sha1new=f20fb51d33212d1357790ae0c435feb5a22e68d4" released="2020-01-22">
+        <manifest-digest sha256new="UTIKPDVXYB5Q7FUFI6XZ6VN5ZZCPIMO232DLD77BPADKVNYVZTAQ"/>
+        <archive extract="upx-3.96-i386_linux" href="https://github.com/upx/upx/releases/download/v3.96/upx-3.96-i386_linux.tar.xz" size="485836" type="application/x-xz-compressed-tar"/>
+      </implementation>
+      <implementation arch="Linux-unknown" id="sha1new=5c4c270b6627c739b779446b71bfec313db8ca28" released="2020-01-22">
```

Comment:
This should probably be `arch="Linux-armv6l"`.

Reply:
I changed it